e2e/snapshot.go

```diff
@@ -53,7 +53,7 @@ var _ = ginkgo.Describe("SPDKCSI-SNAPSHOT", func() {
 				deployTestPod()
 				defer deleteTestPod()
 
-				err := waitForTestPodReady(f.ClientSet, 3*time.Minute)
+				err := waitForTestPodReady(f.ClientSet, 3*time.Minute, "spdkcsi-test")
```

Comment:
This here should be `spdkcsi-test-snapshot2`

Reply:
done